kupiekota/templates/base.html

```diff
@@ -26,6 +25,7 @@
             href="{% url 'posts:for_user' username=user.username %}">
             {{user.username}} Oferty
           </a>
+          <a class="nav-item nav-link" href="{% url 'usermsg:userboxlist' username=request.user.username %}">Wiadomości</a>
```

Comment:
Nie widzę żebyś potem w tym widoku wykorzystywał to `username`. Ten aparametr jest rzeczywiście potrzebny?